addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -1054,6 +1094,26 @@ const actions = {
             chatWindowInitialScrollTops: newChatWindowInitialScrollTops,
         });
     },
+    /**
+     * @param {Object} state
+     * @param {string} composerLocalId
+     * @param {string} textInputContent
+     * @param {integer} textInputCursorStart
+     * @param {integer} textInputCursorEnd
+     */
+    saveComposerContent(
+        { state },
+        composerLocalId,
+        textInputContent,
+        textInputCursorStart,
+        textInputCursorEnd,
```

Comment:
I prefer grouping `textInput*` in an object:
```js
{
    textInputContent,
    textInputCursorStart,
    textInputCursorEnd,
}
```

Reply:
Done (I guess?)